_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -54592,6 +54587,7 @@
 	pixel_y = 2;
 	req_access_txt = "9"
 	},
+/mob/living/carbon/monkey,
```

Comment:
👀

Reply:
He moves....

Reply:
Look man i genuinely don't know how this occured, i don't remember ever touching a single monkey in either the code or the map.

Reply:
Probably just map merger voodoo